TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/claw/Finger.java

```diff
@@ -0,0 +1,43 @@
+package org.firstinspires.ftc.teamcode.subsystems.claw;
+
+import com.qualcomm.robotcore.hardware.Servo;
+
+class Finger {
+
+    private final double fingerOpenPosition;
+    private final double fingerClosePosition;
+    private final Servo fingerServo;
```

Comment:
rename, because you are in the Finger class, consider renaming to `servo` or `servoMotor`

Reply:
nah, its fine (: